### DISCUSSION TITLE
feeds depending on tech stack for developers

### DISCUSSION THREAD

**Discussion by Abdulali97** [dailydotdev/daily#852] [Feature Requests]
As a web developer, it's  hard to keep updated with new features, and the changelog for each technique I use. 

I think it will be a good idea if there was a section for developers to choose the tech stack they use for hopes or work.
 and get news and feeds from the main source, not any source
such as :
 
-- Angular 
  get new features updated, changelog, etc 
--TypeScript
 --same 
.
.
.
. 
etc 
 I additions to that 
 daily dev   "discord bot  "  also a great idea

**Comment by idoshamun** [comment]
Hi @Abdulali97,

Thanks for the suggestion!
This makes a perfect sense, I'll share it internally.
